deps/Chakra.ICU/Chakra.ICU.Build.props

```diff
@@ -36,6 +36,9 @@
         _CRT_SECURE_NO_DEPRECATE;
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
+
+      <!-- Some ICU files use embedded UTF-8 -->
```

Comment:
FWIW, as of ICU v59, all of the source files are supposed to be encoded as UTF-8.

Reply:
Yeah, it makes more sense in Chakra.ICU.Build.props anyways.